### ISSUE TITLE
Add custom command names instead of method name

### ISSUE THREAD

**Issue by dbarwikowski** [joncloud/thor_net#23]

**Comment by joncloud** [comment]
Hey @dbarwikowski ,

Are you looking to provide an alternative way to specify a command so that the name isn't reflected as the method name? If so, would something like the following work to allow opting in and providing alternatives?

```csharp
[Alias("Complex")]
public void This_Is_A_Really_Complex_Name();
```

**Comment by dbarwikowski** [comment]
Hey @joncloud ,
This is exactly what I'm looking for!

I was going to make a PR but I don't have time lately

**Comment by joncloud** [comment]
This will likely be available within the next month in `2.0.0`.
